drivers/overlay/overlay.go

```diff
@@ -441,6 +442,15 @@ func (d *Driver) create(id, parent string, opts *graphdriver.CreateOpts) (retErr
 		return err
 	}
 	if lower != "" {
+		lowers := strings.Split(lower, ":")
+		newpath := path.Join(d.home, lowers[0])
```

Comment:
This will fail if the parent layer is in a read-only store.  Try `newpath := d.dir(lowers[0])` instead.